app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt

```diff
@@ -662,12 +670,16 @@ class ProfileProgressFragmentTest {
         targetViewId = R.id.view_all_text_view,
         stringToMatch = "View All"
       )
+      val recentlyPlayedActivityParams = RecentlyPlayedActivityParams
+        .newBuilder()
+        .setProfileId(ProfileId.newBuilder().setInternalId(internalProfileId).build())
+        .setActivityTitle(RecentlyPlayedActivityTitle.STORIES_FOR_YOU)
+        .build()
       clickProfileProgressItem(itemPosition = 0, targetViewId = R.id.view_all_text_view)
-      intended(hasComponent(RecentlyPlayedActivity::class.java.name))
       intended(
-        hasExtra(
-          RecentlyPlayedActivity.RECENTLY_PLAYED_ACTIVITY_INTERNAL_PROFILE_ID_KEY,
-          internalProfileId
+        allOf(
```

Comment:
Tests to check RecentlyPlayedActivity open from View All is here as mentioned above

Reply:
Thanks, but are we also testing for the other activity title possibility? I think a test still needs to be added for that case.

Reply:
Added test case for same

Reply:
Thanks--I think this can be marked as resolved then.

Reply:
(Marking as resolved).